db/migrate/20220902150227_create_regular_payments.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRegularPayments < ActiveRecord::Migration[7.0]
+  def change
+    create_table :regular_payments, id: :uuid do |t|
```

Comment:
`regular_payments` or `regular_transactions`?

In isolation, I think `RegularPayment` makes more sense and models the domain accurately and clearly.

However, given we already have `TransactionType`, maybe `RegularTransaction` would be more helpful in the context of the application?

What do other people think?

Reply:
We have had a discussion on this already as a team when looking at the first basic ticket and decided to use the regular_transactions `RegularTransaction` naming. That naming has also been used in the [related CFE PR](https://github.com/ministryofjustice/check-financial-eligibility/pull/1076)